layouts/error.vue

```diff
@@ -25,18 +341,78 @@ export default {
       otherError: 'An error occurred',
     };
   },
-  head() {
-    const title =
-      this.error.statusCode === 404 ? this.pageNotFound : this.otherError;
-    return {
-      title,
-    };
-  },
+mounted() {
+  anime({
+    targets: '.row svg',
+    translateY: 10,
+    autoplay: true,
+    loop: true,
+    easing: 'easeInOutSine',
+    direction: 'alternate'
+  });
+
+  anime({
+    targets: '#zero',
+    translateX: 10,
+    autoplay: true,
+    loop: true,
+    easing: 'easeInOutSine',
+    direction: 'alternate',
+    scale: [{value: 1}, {value: 1.4}, {value: 1, delay: 250}],
+    rotateY: {value: '+=180', delay: 200},
+  });
+
+}
 };
 </script>
 
 <style scoped>
-h1 {
-  font-size: 20px;
+
+body
+{
+  background-color:#141019;
+  background-color: radial-gradient(at 50% -20%, #908392, #0d060e) fixed;
+}
+
+
+#handboy
```

Comment:
Please, apply this style guide: https://docs.ckan.org/en/ckan-2.7.3/contributing/css.html

Reply:
and avoid id selectors